CMakeLists.txt

```diff
@@ -240,12 +254,19 @@ elseif(NOT DEFINED LIBRARY_INSTALL_DIR)
 endif()
 message(STATUS "LIBRARY INSTALL DIR: ${LIBRARY_INSTALL_DIR}")
 
+# on windows, since the plugins are dlls, they need to go into the executable folder
```

Comment:
I don't quite get this comment. Why would this be the case? The plugins can go to the plugin directory along with any dependent DLLs for them. As long as it's on the path and set as the plugin folder, it'll work no?

Reply:
vcpkg forbids dlls to show up anywhere inside the lib folder. Stupid policy but

Reply:
Hmm how do you mean? Is that written down somewhere?

Reply:
Not sure but you can see it in the CI logs e.g. https://dev.azure.com/vcpkg/public/_build/results?buildId=92172&view=logs&jobId=7922e5c4-0103-5f8f-ad17-45ce9bb98e80&j=7922e5c4-0103-5f8f-ad17-45ce9bb98e80&t=66eef212-08d6-5e6c-a560-7d01149d0188

Reply:
I imagine all the binaries produced should end up in the bin folder. The lib folder is probably intermediate or dependent libs for the build process.

Reply:
The plugins all end up in lib\csound\plugins64-6.0